src/classes/ElevateCaptureGroupResponse.cls

```diff
@@ -40,6 +41,10 @@ public with sharing class ElevateCaptureGroupResponse {
         return id;
     }
 
+    public String status() {
+        return status;
+    }
```

Comment:
This getter is where we should translate the response Status IF, it actually needs to be "stored" in our domain.

If this value means something to Gift Entry Users, what does it mean?  What can they do now that they couldn't do before?

I'm guessing we need _some_ value here to prevent these records from being prematurely processed by BDI, but the value should be meaningful for our domain.